crates/proto/src/xfer/dns_handle.rs

```diff
@@ -10,17 +10,10 @@
 use futures_util::stream::Stream;
 use tracing::debug;
 
-use crate::{
-    error::*,
-    op::{Edns, Message, Query},
-    runtime::RuntimeProvider,
-    xfer::{DnsRequest, DnsRequestOptions, DnsResponse, SerialMessage},
-};
-
-// TODO: this should be configurable
-// > An EDNS buffer size of 1232 bytes will avoid fragmentation on nearly all current networks.
-// https://dnsflagday.net/2020/
-const MAX_PAYLOAD_LEN: u16 = 1232;
+use crate::error::ProtoError;
+use crate::op::Query;
+use crate::runtime::RuntimeProvider;
+use crate::xfer::{DnsRequest, DnsRequestOptions, DnsResponse, SerialMessage};
```

Comment:
Is there a project style for imports? This is the way I'd write these imports (I guess it's the ["rustls style"](https://github.com/rustls/rustls/blob/main/CONTRIBUTING.md#imports)) but in other PRs (like https://github.com/hickory-dns/hickory-dns/pull/3158 as one example) I see new code using the more nested approach you're adjusting here.

Probably we should make a call one way or the other for new code and document it somewhere?

Reply:
I think we made a decision at some point to adopt the nesting style but did not do an at once sweep to adjust all usage at once, so now some modules are in one style and others in the other style and I mostly try to keep style consistent within the module.

Reply:
Ok, I'll see about getting everyone on the same page and the guidance written down separate from this PR.